src/v3/src/hooks/useOnSubmit.ts

```diff
@@ -194,9 +194,11 @@ export const useOnSubmit = (): (options: OnSubmitHandlerOptions) => Promise<void
     // Required to prevent auth-js from clearing sessionStorage and breaking interaction code flow
     payload.exchangeCodeForTokens = false;
     if (step === 'cancel') {
-      authClient?.transactionManager.clear({ clearIdxResponse: false });
```

Comment:
I think we need to keep this as is to not cause regression, see: https://github.com/okta/okta-signin-widget/blob/master/src/v2/controllers/FormController.ts#L165

However, I think you can add another call to `clear` (without the param) within the `isOauth2Enabled` condition (this is what v2 does as well).

Reply:
I see, thanks. Yeah I changed this because the re-enabled test is failing in CI but not locally so I was trying to change some things around to potentially diagnose what the issue is because the error is now not the same as the original reason this test was failing (which was the `recoveryToken` not being cleared)